android/app/src/main/res/layout/fragment_practice.xml

```diff
@@ -31,17 +31,41 @@
             app:layout_constraintTop_toTopOf="parent"
             tools:text="M" />
 
-        <Button
-            android:id="@+id/next_button"
-            android:layout_width="70sp"
+        <LinearLayout
+            android:layout_width="wrap_content"
             android:layout_height="0dp"
-            android:onClick="@{() -> practiceViewModel.onNext()}"

```

Comment:
It is the best way to call the logic

Reply:
Tried (previous commit), got strange behaviour (when returning from help menu, a hint was being invoked)